doc/manual/manual.tex

```diff
@@ -4745,6 +4745,26 @@ \subsubsection{Using tracer particles}
 \item If one selects a tracer of a particular id, an output of the pressure and temperature at its current position generated by the \texttt{pT path} property over time allows for the creation of a pressure-temperature curve of a certain piece of rock. This property is interesting in many lithosphere to crustal scale models, because it is determining the metamorphic reactions that happen during deformation processes (e.g. in a subduction zone).
 \end{itemize}
 
+\paragraph{Using active tracers}
+In the examples above, tracers properties passively track distinct model properties.  These tracer properties, however, may also be used to actively influence the model as it runs.  For instance, a composition-dependent material model may use a particle's initial composition rather than an advected compositional field.  
+A slightly modified version of the active-composition cookbook (\url{cookbooks/composition-active.prm}:) illustrates how to use 'active tracers' in this manner.
+
+This cookbook (\url{cookbooks/composition-active-tracers.prm}:) modifies two sections of the input file.  First, tracers are added under the \texttt{Postprocess} section: 
+
+\lstinputlisting[language=prmfile]{cookbooks/composition-active-tracers/tracers.part.prm.out}
+
+In this case, the tracked tracer properties are \texttt{velocity} and \texttt{initial composition}.  In order to use the tracer initial composition value to modify the flow through the material model, we now modify the \texttt{Composition} section:
+
+\lstinputlisting[language=prmfile]{cookbooks/composition-active-tracers/composition.part.prm.out}
+ 
+In comparison to \url{cookbooks/composition-active.prm}, the \texttt{Composition} section now contains variables that specify the \texttt{Compositional field method} and \texttt{Mapped particle properties}.  The \texttt{Compositional field method} parameter may set to either \texttt{field} or \texttt{particles}, where \texttt{field} specifies that compositional fields are tracked by advected fields (default method).  
+
+In this case, we specify that the compositional fields are in fact tracked by particles.  In order for the material model to access information stored on the particles, however, we also need to specify that the initial composition of each particle is mapped to a new field created at each time step for use in the material model.  The name of this field will still correspond to those specified in \texttt{set Names of fields}.  
+
+The initial composition is mapped to a new field with the general phrase \texttt{field_name: initial field_name}, where \texttt{field_name} corresponds to \texttt{upper} or \texttt{lower}.  Notably, the order in which field names are defined in \texttt{set Names of fields} will determine the compositional field index number referenced in a material model.  However, it is recommended for consistency that mapped properties are listed in the same order as in \texttt{Set Mapped particle properties} . 
```

Comment:
Maybe change the last sentence to : 
Therefore, it is recommended for consistency that mapped properties are listed in the same order as in \texttt{Set Names of fields}. ?